tests/fragments/test_non_primitive_body.proto

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2021  Google LLC
```

Comment:
Is this file used? If yes, can you please explain  how (I could not find any referrences to it from other parts of this PR

Reply:
It's used in the fragment tests. For each proto in this directory, we generate a GAPIC library and run its generated unit tests. This stresses both the templates and the actual logic of the generated surface. See noxfile.py:fragment for more detail on how the fragment tests work.
This particular fragment is the minimum required reproduction for the issues that compute was having.